Diary/Model/DTO/DiaryData.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  DiaryData.swift
+//  Diary
+//
+//  Created by Dasan, kyungmin on 2023/08/28.
+//
+
+struct DiaryData: Decodable {
+    let title: String
+    let body: String
+    let createdDate: Int
+    
+    private enum CodingKeys: String, CodingKey {
+        case title
+        case body
+        case createdDate = "created_at"
+    }
```

Comment:
```suggestion
    }
    func toDomain() -> Diary {
    return Diary(
      title: self.title,
      body: self.body,
      createdDate: DateFormatter().dateString(
        from: Date(timeIntervalSince1970: Double(self.createdDate)),
        at: Locale.current.identifier,
        with: "MMMMddyyyy")
    )
  }
```
이건 그냥 예시입니다~ 저 같은 경우 dto와 entity사이에 요런 매핑하는 메서드를 활용하기도 합니다~ 정답은 없습니다!!

Reply:
좋은 방법인 것 같아 반영하였습니다.😊